lib/logflare/utils.ex

```diff
@@ -0,0 +1,16 @@
+defmodule Logflare.Utils do
```

Comment:
We're only using this in one piece of the code and it's calling a library directly, I don't see the need to have a utils library for this.